src/vm/appdomain.hpp

```diff
@@ -1323,7 +1326,9 @@ class BaseDomain
     OBJECTHANDLE CreateDependentHandle(OBJECTREF primary, OBJECTREF secondary)
     {
         WRAPPER_NO_CONTRACT;
```

Comment:
This isn't a wrapper anymore - can we use `::CreateDepehdentHandle`'s contract here?